doc/whats_new/v0.23.rst

```diff
@@ -136,6 +136,12 @@ Changelog
   samples in the training set. :pr:`14516` by :user:`Johann Faouzi
   <johannfaouzi>`.
```

Comment:
In line 26 of the what's new, you can add the following:

```python
- :class:`ensemble.BaggingClassifier` and :class:`ensemble.BaggingRegressor`. |Fix|
```

Since we are not sharing the random state anymore, we can have a slight change of score as the example in the `ExtraTreeRegressor` docstring.

Reply:
Thanks a lot. How about to add `IsolationForest` too?
```
- :class:`ensemble.BaggingClassifier`, :class:`ensemble.BaggingRegressor`,  and :class:`ensemble.IsolationForest`. |Fix|
```